### ISSUE TITLE
Accessibility:MAS42A: More colors in high contrast theme

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/vscode#17178]
VS Code Version:1.6.1
OS Version:1607
**Pre-Requisite:**
1.Open visual studio code editor.
2.Using "Ctrl+Shift+P" key Open the command Pallete.
3.Type Theme, Navigate to Preferences: Color Theme and Hit Enter.
4.Navigate to High Contrast and Hit Enter

**Steps to Reproduce:**
1.Open visual studio code editor.
2. Using "Ctrl+Shift+E", key navigate to the Explorer button.
3. Using Tab Arrive at "Open folder" and Hit Enter.
4. Select the Node Modules folder from your Computer.
5. Using Tab, Navigate to a file you want to view and Hit Enter.
6. Visually observe the how Variables, Functions, Modules, Loop Statements , Exceptions, function Names and Variables Names etc are differentiated in High Contrast Theme.

**Expected Result:**
There should be visual  Color Differentiation between Functions and Loops Statements or Exceptions of code snippets in High Contrast Theme.
**Actual Result:**
High Contrast Theme:
1. All Functions, Loops, Modules, Exceptions are defined in Blue color in High Contrast.
2. All Variable Names, Functions Names are White in Color in High Contrast Theme. 
3. Statements are defined in Bolden Orange color

Dark+(Default Dark):
1. Functions and Modules are defined in Blue Color.
2. Loops and Exceptions are defined in Pink Color.
3. Variable Names are defined in Sky Blue Color.
4. Statements are defined in Orange color
4. Function Names are defined in Yellow Color.

**User Impact:**
Low Vision users will not be able easily understand the areas of code snippets without visual color differentiation.

![example 1_there is no visual color differentiation between functions and loops statemens or exceptions of code snippets in high contrast theme](https://cloud.githubusercontent.com/assets/22956130/21185531/0d13dd66-c237-11e6-9946-6a2437666bf3.jpg)
![example 2_there is no visual color differentiation between functions and loops statemens or exceptions of code snippets in high contrast theme](https://cloud.githubusercontent.com/assets/22956130/21185533/0e136c4a-c237-11e6-8310-275d1a349849.jpg)

**Comment by WiproPSAccTest** [comment]
For a color blindness or low vision people color differentiation should be present. color differentiation makes it easy for them to differentiate different areas of the code.

**Comment by WiproPSAccTest** [comment]
Title has been changed to a common format to perform data population.

**Comment by isidorn** [comment]
I can look into this. Assigning to me

**Comment by isidorn** [comment]
Now we are using all the dark+ token colors in high contrast theme
